set.mm

```diff
@@ -274564,27 +274561,26 @@ the same dimension over the same (nonzero) ring.  (Contributed by AV,
     $( An associative algebra is a left module.  (Contributed by Mario
        Carneiro, 5-Dec-2014.) $)
     assalmod $p |- ( W e. AssAlg -> W e. LMod ) $=
-      ( vz vx vy casa wcel clmod crg csca cfv ccrg w3a cv cvsca cmulr wceq wral
-      co cbs eqid wa isassa simplbi simp1d ) AEFZAGFZAHFZAIJZKFZUEUFUGUILBMZCMZ
-      ANJZRDMZAOJZRUJUKUMUNRULRZPUKUJUMULRUNRUOPUADASJZQCUPQBUHSJZQCDUQULUNUHUP
-      ABUPTUHTUQTULTUNTUBUCUD $.
+      ( vz vx vy casa wcel clmod crg wa cv cvsca cfv co wceq cbs wral csca eqid
+      cmulr isassa simplbi simpld ) AEFZAGFZAHFZUCUDUEIBJZCJZAKLZMDJZASLZMUFUGU
+      IUJMUHMZNUGUFUIUHMUJMUKNIDAOLZPCULPBAQLZOLZPCDUNUHUJUMULABULRUMRUNRUHRUJR
+      TUAUB $.
 
     $( An associative algebra is a ring.  (Contributed by Mario Carneiro,
        5-Dec-2014.) $)
     assaring $p |- ( W e. AssAlg -> W e. Ring ) $=
-      ( vz vx vy casa wcel clmod crg csca cfv ccrg w3a cv cvsca cmulr wceq wral
-      co cbs eqid wa isassa simplbi simp2d ) AEFZAGFZAHFZAIJZKFZUEUFUGUILBMZCMZ
-      ANJZRDMZAOJZRUJUKUMUNRULRZPUKUJUMULRUNRUOPUADASJZQCUPQBUHSJZQCDUQULUNUHUP
-      ABUPTUHTUQTULTUNTUBUCUD $.
+      ( vz vx vy casa wcel clmod crg wa cv cvsca cfv co wceq cbs wral csca eqid
+      cmulr isassa simplbi simprd ) AEFZAGFZAHFZUCUDUEIBJZCJZAKLZMDJZASLZMUFUGU
+      IUJMUHMZNUGUFUIUHMUJMUKNIDAOLZPCULPBAQLZOLZPCDUNUHUJUMULABULRUMRUNRUHRUJR
+      TUAUB $.
+  $}
 
+  ${
     assasca.f $e |- F = ( Scalar ` W ) $.
-    $( An associative algebra's scalar field is a commutative ring.
-       (Contributed by Mario Carneiro, 7-Jan-2015.) $)
-    assasca $p |- ( W e. AssAlg -> F e. CRing ) $=
-      ( vz vx vy casa wcel clmod crg ccrg w3a cv cvsca cfv wceq cbs wral eqid
-      co cmulr wa isassa simplbi simp3d ) BGHZBIHZBJHZAKHZUFUGUHUILDMZEMZBNOZTF
-      MZBUAOZTUJUKUMUNTULTZPUKUJUMULTUNTUOPUBFBQOZREUPRDAQOZREFUQULUNAUPBDUPSCU
-      QSULSUNSUCUDUE $.
+    $( An associative algebra's scalar field is a ring.  (Contributed by Mario
```

Comment:
```suggestion
    $( The ring of scalars of an associative algebra is a ring.  (Contributed by Mario
```

Reply:
I see that "field" might be a bit confusing, so I took the suggestion, and then since "ring of" is redundant that was removed/reworded.